components/clarity-vscode/server/src/common.ts

```diff
@@ -19,6 +19,7 @@ export function initConnection(
   connection.onInitialize((params) => {
     try {
       initializationOptions = JSON.parse(params.initializationOptions);
+      params.initializationOptions = initializationOptions;
```

Comment:
fix options parsing

Reply:
Could skip the `initializationOptions` let entirely.

Reply:
Reused here: https://github.com/hirosystems/clarinet/blob/ce6bf85ca18d48940210a9c4449a48c69a4b5c53/components/clarity-vscode/server/src/common.ts#L68